docs/content/slides/pack-your-bags/index.md

```diff
@@ -61,49 +61,356 @@ name: introductions
   </div>
 </div>
 
+---
+name: kickoff
+class: center, middle
+# First A Quick Demo!
+
 ---
 name: cnab
+# What's a CNAB???
+
+---
+class: center, middle
+
+# Let's Answer That With A Story!
 
-# What is CNAB?
+---
+class: center, middle
 
-Quick demo of 2 min demo of porter doing a porter install (azure + helm)
-Runs while we talk (azure-mysql-wordpress or a quickstart from Brian)
+# The Cast
 
-Break up our talk 1-20 + 23
+---
+class: center, middle
 
-???
-Explain why a spec is necessary
+# You!
+.center[
+  ![you, a developer](/images/pack-your-bags/you-a-developer.jpg)
+]
+
+---
+class: center, middle
+
+# Your friend!
+.center[
+  ![your friend, a computer user](/images/pack-your-bags/your-friend-a-user.jpg)
+]
+
+---
+class: center, middle
+
+# Your friend!
+.center[
+  ![your friend, a computer user](/images/pack-your-bags/your-friend-a-user.jpg)
+]
+
+---
+class: center, middle
+
+# Your App!
+.center[
+  ![it's the journey that matters](/images/pack-your-bags/mcguffin.png)
+]
+
+---
+class: center, middle
+
+# Your Fans!
+.center[
+  ![trending](/images/pack-your-bags/your-fans.jpg)
+]
+
+---
+class: center, middle
+
+# Act One!
+
+---
+class: center, middle
+
+# You Built an App
+
+.center[
+  ![you again](/images/pack-your-bags/you-a-developer.jpg)
+  ![it's the journey that matters](/images/pack-your-bags/mcguffin.png)
+]
+
+---
+class: center, middle
+
+# It Runs Happily In The Cloud
+
+---
+# ....your cloud
+.center[
+  ![that's a bingo](/images/pack-your-bags/cloud-bingo.png)
+]
+
+---
+class: center, middle
+
+# Act Two
+
+---
+class: center, middle
+
+# Your Friend Wants To Run It!
+.center[
+  ![your friend, a computer user](/images/pack-your-bags/your-friend-a-user.jpg)
+]
+
+---
+class: center, middle
+
+# How exciting! 
+
+--
+
+# So you write extensive docs
+.center[
+  ![you fight for the users](/images/pack-your-bags/scroll-of-truth.png)
+]
+
+---
+class: center, middle
+
+# You are no longer friends...
+
+.center[
+  ![you fight for the users](/images/pack-your-bags/Spongebob-patrick-crying.jpg)
+]
+
+.footnote[http://vignette3.wikia.nocookie.net/spongebob/images/f/f0/Spongebob-patrick-crying.jpg/revision/latest?cb=20140713205315]
+
+---
+class: center, middle
+
+# So you work together...
+
+.center[
+  ![pair programming](/images/pack-your-bags/working-together.jpg)
+]
+
+---
+class: center, middle
+
+# Then you help a few more people...
+
+.center[
+  ![go team](/images/pack-your-bags/go-team.jpg)
+]
+
+---
+class: center, middle
+
+# Suddenly McGuffin has FANS!
+
+.center[
+  ![all the github stars!!!](/images/pack-your-bags/your-fans.jpg)
+]
+
+---
+class: center, middle
+
+# Everyone do this is not going to cut it...
```

Comment:
Did you mean "This is too damn hard"? or "This won't scale" or ? What's there doesn't read well